src/core/useTrackBindingPlugin.js

```diff
@@ -62,7 +63,8 @@ export class TrackBindingPlugin {
             return;
         }
 
-        let elem = event.target || event.srcElement;
+        var originalElem = event.target || event.srcElement;
```

Comment:
I believe we do need that variable as elem becomes reassigned and I want to be able to track data on the original element that was clicked. Happy to change to let